packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -52,8 +52,10 @@ import {
 } from './actionHandlers';
 import VirtualScroll from '../utils/components/VirtualScroll';
 import { SELECTION_CELL_ID } from './columns';
+import { onShiftSelectHandler } from './actionHandlers/selectHandler';
```

Comment:
Move this import to other `actionHandlers` import.

Reply:
**[DONE]** Ohh, right, thanks